test/e2e/pages/allocation-details.js

```diff
@@ -47,7 +48,10 @@ class AllocationDetailsPage extends Page {
       },
       date: {
         selector: this.fields.date,
-        value: format(faker.date.future(), 'yyyy-MM-dd'),
+        value: format(
+          faker.date.future(1, chrono.en.parseDate('tomorrow')),
```

Comment:
Would [`startOfTomorrow`](https://date-fns.org/v1.29.0/docs/startOfTomorrow) work to avoid needing to import two libraries?

Reply:
👍